lib/broadway.ex

```diff
@@ -952,6 +952,27 @@ defmodule Broadway do
     Topology.topology(broadway)
   end
 
+  @doc """
+  Returns all running Broadway names.
+
+  It's important to notice that no order is guaranteed.
+  """
+  def all do
+    Enum.reduce(:persistent_term.get(), [], fn key_value, names ->
+      case key_value do
+        {{Broadway, name}, %Broadway.Topology.Config{}} ->
+          if Process.whereis(name) do
+            [name | names]
+          else
+            names
+          end
+
+        _ ->
+          names
+      end
+    end)
```

Comment:
It should be cleaner as a comprehension.

```suggestion
    for {{Broadway, name}, %Broadway.Topology.Config{}} <- :persistent_term.get(),
        Process.whereis(name),
        do: name
```